Parsers.Common/VisualStudio/Projects/IProjectParser.cs

```diff
@@ -6,6 +6,8 @@ namespace Skyline.DataMiner.CICD.Parsers.Common.VisualStudio.Projects
 {
     internal interface IProjectParser
     {
+        ProjectType GetProjectType();
```

Comment:
Project Type is associated more with the 'type' of project you can select in the creation wizard of visual studio when adding new projects.
I suggest to use ProjectStyle GetProjectStyle(); to better differentiate between the 'style: SDK/Legacy' and 'type' of the projects.

Reply:
Done